src/lib/utils.js

```diff
@@ -0,0 +1,164 @@
+const cbor = require("cbor")
```

Comment:
Avoid using `require` because we can use `ES modules` standard as we are using in other files